client.py

```diff
@@ -386,6 +387,7 @@
 	ewcfg.cmd_adorn: ewcosmeticitem.adorn,
 	ewcfg.cmd_dedorn: ewcosmeticitem.dedorn,
 	ewcfg.cmd_create: ewkingpin.create,
+	ewcfg.cmd_exalt: ewkingpin.exalt,
```

Comment:
we only need the command once

Reply:
If you’re referring to how !exalt is essentially a redlined version of !create, with the only difference being a predefined item, then I suppose that’s true, but for now I think it’s fine. It feels more sophisticated that just spamming !create with the same name and description, just with different recipients.